content/_data/changelogs/weekly.yml

```
@@ -7517,6 +7517,47 @@
   # pull: 4704 (PR title: Add proper title for setup wizard)
   # pull: 4705 (PR title: Update at-since up to May 3, 2020)
 
+- version: '2.236'
+  date: 2020-05-11
+  changes:
+  - type: major bug
+    category: regression
+    pull: 4713
+    issue: 62163
+    authors:
+    - daniel-beck
+    message: |-
+      Make plugin manager work on IE11 again ("regression" in 2.231).
```

```suggestion
      Make plugin manager work on Internet Explorer 11 again (regression in 2.231).
```

I do not think we want to use `"regression"`. It looks to be a bit passive-aggressive, IE is a supported browser:  https://www.jenkins.io/doc/administration/requirements/web-browsers/